src/packages/block/block/workspace/block-workspace-editor.element.ts

```diff
@@ -4,13 +4,16 @@ import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 
 @customElement('umb-block-workspace-editor')
 export class UmbBlockWorkspaceEditorElement extends UmbLitElement {
-	//
+
 	@property({ type: String, attribute: false })
 	workspaceAlias?: string;
 
+	@property({ type: String, attribute: false })
```

Comment:
Hi @rickbutterfield 

I'm so happy that you are getting your hands dirty and getting into the code.

To prevent, too much prop drilling and hard connections, we encourage using the Context API. So in this case you would consume the Block Workspace Context, and observe the name, which should be available via an observable. 

In this way there is a bit of plumbing below in the routes that can be cleaned out :-)

I hope that makes sense :-)
Lets me know what you think and if you have ideas for better DX